### PR TITLE
Change file docstring

### PR DESCRIPTION
**Pull request by RonsenbergVI**
The file docstring was most likely a copy paste from a Bijector submodule file.

### CODE REVIEW

**Comment by RonsenbergVI** [comment]
@googlebot I signed it!

File: tensorflow_probability/python/math/psd_kernels/feature_transformed_test.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ============================================================================
-"""Bijectors Tests."""
+"""Feature transformed Tests."""
```
Comment (ColCarroll) [inline tensorflow_probability/python/math/psd_kernels/feature_transformed_test.py:15]:
```suggestion
"""FeatureTransformed tests."""
```

**Review by ColCarroll — COMMENTED** [review]
Thanks for catching this and making a PR! Mind if we just update the capitalization?

File: tensorflow_probability/python/math/psd_kernels/feature_transformed_test.py (hunk 1 above)
Comment (RonsenbergVI) [reply tensorflow_probability/python/math/psd_kernels/feature_transformed_test.py:15]: sure!

**Review by ColCarroll — APPROVED** [review]

**Comment by ColCarroll** [comment]
Thanks again, @RonsenbergVI !

**Pull request merged**